fhir-audit/src/test/java/com/ibm/fhir/audit/mapper/impl/AuditEventMapperTest.java

```diff
@@ -89,7 +89,7 @@ public void testAuditEventMapper() throws Exception {
+        String expected = "{\"resourceType\":\"AuditEvent\",\"type\":{\"system\":\"http://terminology.hl7.org/CodeSystem/audit-event-type\",\"code\":\"rest\",\"display\":\"Restful Operation\"},\"subtype\":[{\"system\":\"http://hl7.org/fhir/restful-interaction\",\"code\":\"create\",\"display\":\"create\"}],\"action\":\"C\",\"period\":{\"start\":\"2020-12-10T14:35:57.752Z\",\"end\":\"2020-12-10T14:36:02.779Z\"},\"recorded\":\"TIME\",\"outcome\":\"0\",\"outcomeDesc\":\"success\",\"purposeOfEvent\":[{\"coding\":[{\"system\":\"http://terminology.hl7.org/CodeSystem/v3-ActReason\",\"code\":\"PurposeOfUse\",\"display\":\"PurposeOfUse\"}]}],\"agent\":[{\"role\":[{\"coding\":[{\"system\":\"http://terminology.hl7.org/CodeSystem/extra-security-role-type\",\"code\":\"datacollector\",\"display\":\"datacollector\"}]}],\"name\":\"fhir-server\",\"requestor\":true,\"network\":{\"address\":\"192.168.12.28\",\"type\":\"1\"}}],\"source\":{\"site\":\"Location\",\"observer\":{\"display\":\"fhir-server\"},\"type\":[{\"system\":\"http://terminology.hl7.org/CodeSystem/security-source-type\",\"code\":\"4\",\"display\":\"Application Server\"}]},\"entity\":[{\"securityLabel\":[{\"system\":\"http://terminology.hl7.org/CodeSystem/v3-Confidentiality\",\"code\":\"N\",\"display\":\"normal\"}],\"description\":\"FHIR Create request\",\"detail\":[{\"type\":\"FHIR Context\",\"valueBase64Binary\":\"CnsKICAgICJyZXF1ZXN0X3VuaXF1ZV9pZCI6ICI0MTdiMjEwNS1iYzZiLTQ0ZjctOWYyMy00Yzc4NGExN2QyNGIiLAogICAgImFjdGlvbiI6ICJSIiwKICAgICJvcGVyYXRpb25fbmFtZSI6ICJVbml0VGVzdCIsCiAgICAic3RhcnRfdGltZSI6ICIyMDIwLTEyLTEwIDE0OjM1OjU3Ljc1MiIsCiAgICAiZW5kX3RpbWUiOiAiMjAyMC0xMi0xMCAxNDozNjowMi43NzkiLAogICAgImFwaV9wYXJhbWV0ZXJzIjogewogICAgICAgICJyZXF1ZXN0IjogImhvc3RuYW1lIiwKICAgICAgICAicmVxdWVzdF9zdGF0dXMiOiAyMDEKICAgIH0sCiAgICAiZGF0YSI6IHsKICAgICAgICAicmVzb3VyY2VfdHlwZSI6ICJPYnNlcnZhdGlvbiIsCiAgICAgICAgImlkIjogIjBlODM3NDQwLTY0ZWItNGYxNi04NzhlLTEyODc5YTRjZmUyZCIsCiAgICAgICAgInZlcnNpb25faWQiOiAiMSIKICAgIH0sCiAgICAiYmF0Y2giOiB7CiAgICAgICAgInJlc291cmNlc19yZWFkIjogMTAsCiAgICAgICAgInJlc291cmNlc19jcmVhdGVkIjogNSwKICAgICAgICAicmVzb3VyY2VzX3VwZGF0ZWQiOiAyCiAgICB9LAogICAgImV2ZW50X3R5cGUiOiAiZmhpci1jcmVhdGUiLAogICAgImRlc2NyaXB0aW9uIjogIkZISVIgQ3JlYXRlIHJlcXVlc3QiLAogICAgImNsaWVudF9jZXJ0X2NuIjogIiIsCiAgICAiY2xpZW50X2NlcnRfaXNzdWVyX291IjogIldhdHNvbiBIZWFsdGgiLAogICAgImxvY2F0aW9uIjogIkxvY2F0aW9uIgp9\"}]}]}".replace("TIME", now.getValue().format(Instant.PARSER_FORMATTER));
```

Comment:
Please update per my recommendation in AuditEventMapper

Reply:
Done.